sklearn/metrics/classification.py

```diff
@@ -2378,6 +2378,6 @@ def brier_score_loss(y_true, y_prob, sample_weight=None, pos_label=None):
                 np.array_equal(labels, [-1])):
             pos_label = 1
         else:
-            pos_label = y_true.max()
+            pos_label = labels.max()
```

Comment:
This is a separate change. I don't think it belongs in this PR. It is not tested.

Reply:
I will revert and open another pr then